templates/chat.html

```diff
@@ -1,52 +1,68 @@
 {% extends 'index.html' %}
-{% block header %}
-<script src="static/chat.js"></script>
-<script src="static/online.js"></script>
-<script src="static/styles.js"></script>
-{% endblock %}
+<!-- {% block header %}
```

Comment:
remove the old insert tags, because we don't use them anymore

Reply:
they are still there?

Reply:
resolved